install/kubernetes/helm/istio/charts/pilot/values.yaml

```diff
@@ -10,6 +10,12 @@ autoscaleMax: 5
 image: pilot
 sidecar: true
 traceSampling: 1.0
+# specify the applications namespace pilot manages, separated by comma, if not empty,
+# the value must contain the namespace in which Istio control plane located('istio-system' by default);
+# if the field is set to empty, pilot watches all namespaces.
+# example: appNamespace: "istio-system, istio-apps"
+# pilot would watch services, endpoints, pods and nodes in these two namespaces
+appNamespace: ""
```

Comment:
UX: normally this is a list. 

Note that new installer defines the concept of 'environment', where namespaces specify which pilot to use by using a label on the namespace. How does it interact with that ? 

I'm also concerned that this should affect more than pilot - citadel, mixer, etc will need similar things.
Should it be a global setting ?